setup.cfg

```diff
@@ -39,3 +39,10 @@ testing =
 	# local
 	PyQt5; python_version>='3.0'
 	PySide2; python_version=='2.7'
+
+[options.entry_points]
+console_scripts = 
+	cola = cola.__main__:run
+	dag = cola.__main__:run_dag
+	git-cola = cola.__main__:run
+	git-dag = cola.__main__:run_dag
```

Comment:
What about the makefile and the scripts in `bin/`?  Won't these clash? Do we have to inhibit the installation of the `bin/` scripts?

Reply:
One thing that might not be readily apparent is that we currently can run directly from the source tree without any python or build system getting in our way. In other words, I can clone the repo and directly run `./git-cola/bin/git-cola` and it works.

The bin/ scripts currently also handle the Windows installer `$prefix/pkgs/` path where modules get installed. I'm curious if that works -- have you tested generating a win32 installer and setting if the installation works when using the entry points?

Reply:
Yes I do think this kind of competes with the scripts in `bin/`.  But at this point this is is not yet polished enough to do that.

Running straight from the repo is only possible if you already have PyQt5 (or an alternative) available in the *right* Python installation.  Installing through pip could potentially solve that.

I did not consider the Windows installer yet.